src/app/recent_files.cpp

```diff
@@ -100,15 +100,16 @@ QDataStream& operator<<(QDataStream& stream, const RecentFile& recentFile)
 {
     stream << recentFile.filepath;
     stream << recentFile.thumbnail;
-    stream << recentFile.thumbnailTimestamp;
+    stream << (qint64) recentFile.thumbnailTimestamp;
     return stream;
 }
 
 QDataStream& operator>>(QDataStream& stream, RecentFile& recentFile)
 {
     stream >> recentFile.filepath;
     stream >> recentFile.thumbnail;
-    stream >> recentFile.thumbnailTimestamp;
+    const quint64 thumbnailTimestamp(recentFile.thumbnailTimestamp);
+    stream >> thumbnailTimestamp;
```

Comment:
There is compilation error here reported by the appveyor CI
This should be :
`stream >> reinterpret_cast<qint64&>(recentFile.thumbnailTimestamp);`

Reply:
Done!